pkg/client/fake/client.go

```diff
@@ -29,8 +29,7 @@ import (
 	"sync"
 	"time"
 
-	// Using v4 to match upstream
```

Comment:
Did upsream switch to v5?

Reply:
What is meant by upstream?

Reply:
https://github.com/kubernetes-sigs/controller-runtime/pull/2643#discussion_r1446189655

Reply:
Also see the comment in go.mod

> // Using v4 to match upstream

Maybe we should add more details like linking #2643

Reply:
Yup. That's a question for k/k. I'm pretty sure there already were discussions about this. I just don't know where